### PR TITLE
add chipset ite choice

### PR DESCRIPTION
**Pull request by chiachiehlin**
iTE series

### CODE REVIEW

File: ecfwwork/CPGZephyrDocs
Comment (yimjiajun) [inline ecfwwork/CPGZephyrDocs:1]: please remove this file from PR

File: ami_build.sh [hunk 1]
```diff
@@ -72,7 +72,21 @@ function parameters_selection() {
...
+            ;;    
+            "ite")
+                board_info["chipset"]="it"
+                case ${info["series"]} in
+                    "81202")
+                        board_info["series"]="81202"
+                        ;;
+                    "82202")
+                        board_info["series"]="82202"
+                        ;;
```
Comment (yimjiajun) [inline ami_build.sh]: this part is convert to compatible series number if providing 8120X might convert to 81202

so we can remove this statement currently

**Comment by yimjiajun** [comment]
remove  compatible series number if providing 8120X might convert to 81202

File: ami_build.sh
Comment (yimjiajun) [inline ami_build.sh:1]: this part is convert to compatible series number if providing 8120X might convert to 81202

so we can remove this statement currently
@chiachiehlin

File: ami_build.sh [hunk 2]
```diff
@@ -76,12 +76,9 @@ function parameters_selection() {
...
                 board_info["chipset"]="it"
                 case ${info["series"]} in
-                    "81202")
-                        board_info["series"]="81202"
+                    "8120"*)
+                        board_info["series"]="812"
                         ;;
-                    "82202")
-                        board_info["series"]="82202"
-                        ;;
```
Comment (yimjiajun) [inline ami_build.sh]: 這寫部分我們可以拿掉,暫時不用做切換.可以移除它們

File: ecfwwork/CPGZephyrDocs
Comment (yimjiajun) [inline ecfwwork/CPGZephyrDocs:1]: 目前可以移除它,然後再上傳.過後我們再讓你嘗試reset/drop 該commit

**Review by yimjiajun — COMMENTED** [review]
這裏都是要移除的 不用新加東西

**Comment by chiachiehlin** [comment]
already fixed it

**Pull request closed without merging**